composer.json

```diff
@@ -16,19 +16,20 @@
             "name":"Michael Bodnarchuk"
         }
     ],
-    "minimum-stability": "RC",
+    "minimum-stability": "dev",
     "require": {
-        "php": ">=5.6.0 <=8.1 | ~8.1.0",
-        "codeception/lib-innerbrowser": "^1.0",
-        "codeception/codeception": "^4.0"
+        "php": "^8.0",
+        "codeception/codeception": "^5.0.0-alpha2",
+        "codeception/lib-innerbrowser": "^3.0"
     },
     "require-dev": {
         "yiisoft/yii2": "dev-master",
         "yiisoft/yii2-app-advanced": "dev-master",
-        "codeception/verify": "<2",
+        "phpunit/phpunit": "dev-master as 9.5.99",
```

Comment:
Please remove constraint of `phpunit/phpunit` unless you really have dependency on PHPUnit 10.